### PR TITLE
Add Jenkinsfile with build pipeline

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

**Comment by ifosch** [comment]
Now it is getting built @miguelbernadi

**Comment by miguelbernadi** [comment]
+1

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    environment {
+        REPO = "406396564037.dkr.ecr.us-east-1.amazonaws.com/jarvis"
+        TAG = "${GIT_COMMIT}"
+    }
+    stages {
+        stage('Credentials') {
+            steps {
+                sh 'sudo pip install awscli'
+                sh 'aws ecr get-login --region us-east-1 --no-include-email | bash'
```
Comment (miguelbernadi) [inline Jenkinsfile]: This is only needed when we do the push

Comment (ifosch) [reply Jenkinsfile]: Fixed

**Comment by miguelbernadi** [comment]
+1 still

**Comment by miguelbernadi** [comment]
+1 still

**Comment by towanda** [comment]
+1

**Pull request merged**